### PR TITLE
Profile page and implement API

### PR DESCRIPTION
**Pull request by danny237**
image

### CODE REVIEW

File: src/Themes/Profile.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.backgroundHolder {
```
Comment (silwalanish) [inline src/Themes/Profile.scss:1]: Use kebab-case for naming class.
```suggestion
.background-holder {
```

File: src/Pages/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    this.state = {
+      reconstructions: [],
+    };
+  }
+
+  async componentDidMount() {
+    let axios = await getAuthenticatedInstance();
+    try {
+      let resp = await axios.get(
+        `https://threedify-api-staging.herokuapp.com/users/${this.props.user.id}/reconstructions`
```
Comment (silwalanish) [inline src/Pages/Profile.jsx:21]: Store this as a templated constant.
e.g.
```ks
const USER_RECONSTRUCTIONS_API = process.env.API_URL + '/users/{userId}/reconstructions'
```
Then you can use that here like
```js
USER_RECONSTRUCTIONS_API.replace('{userId}', this.props.user.id)
```

File: src/Pages/Profile.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                <p>150</p>
+              </div>
+            </div>
+          </div>
+
+          {/* model section */}
+          <div className='model-section ml-2 col-7'>
+            <h2 className='ml-5 text-center'>Models</h2>
+            <hr />
+            <div className='d-flex ml-5 flex-wrap'>{cards}</div>
```
Comment (silwalanish) [inline src/Pages/Profile.jsx:93]:
```suggestion
            <h3 className='border-bottom text-center'>Models</h3>
            <div className='d-flex flex-wrap'>{cards}</div>
```

File: src/Pages/Profile.jsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+      // maincontainer
+      <div className='col-12'>
+        {/* background holder*/}
+        <div className='background-holder col-12'></div>
+
+        {/* profile section */}
+        <div className='main-content col-12'>
+          <div className='profile-section col-3'>
+            {/* profile img */}
+            <div className='user-profile-pic border border-primary rounded-circle text-center text-primary'>
```
Comment (silwalanish) [inline src/Pages/Profile.jsx:65]:
```suggestion
            <div className='user-profile-pic border border-primary rounded-circle text-center text-primary mb-0'>
```

File: src/Pages/Profile.jsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+              {firstLetter}
+            </div>
+            <h2 className='font-weight-medium'>{fullName}</h2>
+            <h4>{this.props.user.username}</h4>
```
Comment (silwalanish) [inline src/Pages/Profile.jsx:69]:
```suggestion
            <h4 className='font-weight-medium'>{fullName}</h4>
            <p className='text-muted'>{this.props.user.username}</p>
```

File: src/Pages/Profile.jsx [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+            <button className='btn btn-primary col-8 my-4'>Follow</button>
```
Comment (silwalanish) [inline src/Pages/Profile.jsx:71]:
```suggestion
            <button className='btn btn-primary col-8 mb-4'>Follow</button>
```

**Review by silwalanish — APPROVED** [review]

File: src/Pages/Profile.jsx [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+              </div>
+              <div className='info-item d-flex'>
+                <p className='mr-auto'>Views</p>
+                <p>150</p>
+              </div>
+            </div>
+          </div>
+
+          {/* model section */}
+          <div className='model-section ml-2 col-7'>
```
Comment (silwalanish) [inline src/Pages/Profile.jsx:90]:
```suggestion
          <div className='model-section ml-2'>
```

File: src/Themes/Profile.scss [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+  height: 750px;
+  margin-top: -125px;
+  z-index: 1;
+  background-color: map-get($map: $theme-colors, $key: 'white');
+  border-radius: 10px;
+  box-shadow: 0px 10px 15px rgba(0, 0, 0, 0.2);
+}
+
+.model-section {
+  padding: 20px;
```
Comment (silwalanish) [inline src/Themes/Profile.scss:26]:
```suggestion
  padding: 20px;
  max-width: 1098px;
  box-sizing: content-box;
```

**Comment by danny237** [comment]
Looks Good

**Pull request merged**